src/components/transport_manager/include/transport_manager/tcp/network_interface_listener.h

```diff
@@ -1,35 +1,3 @@
-/*
```

Comment:
I believe there should still be some copyright included in the files where they were completely removed. Was the intention to replace the Xevo copyrights with Ford Motor Company?

Reply:
I was just asked to "remove" our copyright notices. Since the code is owned by Ford team, I think it's up to them to put their copyright notice (or something other).